### ISSUE TITLE
Integration of asec and acsc

### ISSUE THREAD

**Issue by normalhuman** [sympy/sympy#14329]
These are admittedly obscure functions, but they can be integrated. 
```
>>> integrate(asec(x), x)
Integral(asec(x), x)
>>> integrate(acsc(x), x)
Integral(acsc(x), x)
```
SymPy can do it with a rewrite in terms of more familiar acos and asin. (Recall asec(x) is acos(1/x) and acsc(x) is asin(1/x).)
```
>>> integrate(asec(x).rewrite(asec, acos), x)
x*acos(1/x) - Piecewise((acosh(x), Abs(x**2) > 1), (-I*asin(x), True))
>>> integrate(acsc(x).rewrite(acsc, asin), x)
x*asin(1/x) + Piecewise((acosh(x), Abs(x**2) > 1), (-I*asin(x), True))
```
Should this rewrite be done by `integrate` automatically?

**Comment by jksuom** [comment]
> Should this rewrite be done by integrate automatically?

That will be done by `rischintegrate` once integration of trigonometric functions and their inverses is implemented. But I think that there could be a rewrite operation common for all integrators.

**Comment by avishrivastava11** [comment]
@normalhuman @jksuom  In-fact, I suggest we add a `trig_integrate` method to `Integral class` specifically to handle such trig integrals. What do you say Sir?

**Comment by jksuom** [comment]
I don't think that adding another method to `Integral` would be a good idea. How would a user know when to use such a method. The `doit` method should be able to handle ordinary integrals (including improper integrals) when they exist.

**Comment by iammosespaulr** [comment]
https://github.com/sympy/sympy/issues/19983#issuecomment-677694404

**Comment by asmeurer** [comment]
I think that issue can be closed as a duplicate of this one.

**Comment by eagleoflqj** [comment]
Fixed by #19993
